composer.json

```diff
@@ -33,5 +33,10 @@
     "exclude-from-classmap": [
       "/Tests/"
     ]
+  },
+  "extra": {
+    "branch-alias": {
+      "dev-master": "1.0-dev"
```

Comment:
I suggest using ``1.0.x-dev``, which is the actual branch alias.

``1.0-dev`` is not a branch version, but a tag version. Composer has a special handling to accept it in branch aliases by rewriting it, but it confuses people in some cases (``composer require 1.0-dev`` would not find any matching package for instance, as it should be ``composer require 1.0.x-dev``)